src/model/ExternalFile.cpp

```diff
@@ -206,12 +209,17 @@ namespace model {
 
   }  // namespace detail
 
-  boost::optional<ExternalFile> ExternalFile::getExternalFile(const Model& model, const std::string& filename) {
+  boost::optional<ExternalFile> ExternalFile::getExternalFile(const Model& model, const std::string& filename, bool copy) {
     path p = toPath(filename);
     if (!p.has_filename()) {
       return boost::none;
     }
-    std::string s = toString(p.filename());
+    std::string s;
+    if (copy) {
+      s = toString(p.filename());
+    } else {
+      s = filename;
+    }
```

Comment:
I'm not certain about this. Do we allow several files named the same in different directories?
I guess that's fine, since it's filePath() that's used in the FT.

https://github.com/NREL/OpenStudio/blob/42683d615dd470d90390175c1fdc884039dbd767/src/energyplus/ForwardTranslator/ForwardTranslateScheduleFile.cpp#L68